pkg/util/client.go

```diff
@@ -27,3 +29,14 @@ func GetClientSet(file string) (clientset.Interface, error) {
 
 	return clientset.NewForConfig(restConfig)
 }
+
+// SplitSliceString is a helper function to handle StringSliceFlag containing mutliple values
```

Comment:
> pkg/util/client.go:33:79: `mutliple` is a misspelling of `multiple` (misspell)

Also while you're there, a better name for this might be `Resplit` or `SplitStringSlice` since it's a String Slice, not a Slice String.

Maybe could also take a configurable delimiter like Split does, but that might be premature optimization.